sdcm/nemesis.py

```diff
@@ -423,6 +423,32 @@ def disrupt_restart_then_repair_node(self):  # pylint: disable=invalid-name
         self.target_node.wait_jmx_up()
         self.repair_nodetool_repair()
 
+    def disrupt_resetlocalschema(self):  # pylint: disable=invalid-name
+        self._set_current_disruption('ResetLocalSchema %s' % self.target_node)
+
+        result = self.target_node.run_nodetool(sub_cmd='help', args='resetlocalschema')
+        if 'Unknown command resetlocalschema' in result.stdout:
+            raise UnsupportedNemesis("nodetool doesn't support resetlocalschema")
+
+        rlocal_schema_res = self.target_node.follow_system_log(patterns=["schema_tables - Schema version changed to"])
+        self.target_node.run_nodetool("resetlocalschema")
+        assert list(rlocal_schema_res), "Schema version has not been recalculated"
+
+        # Waiting 60 sec: this time is defined by Tomasz
+        self.log.debug("Sleep for 60 sec: the other nodes should pull new version")
+        time.sleep(60)
+
+        # Check schema version on the nodes - expected the same version on the all nodes
```

Comment:
no need to check that, we run health check validator that verifies that schema is the same on all nodes